services/QuillLMS/spec/mailers/previews/rollup_mailer_preview.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+  class RollupMailerPreview < ActionMailer::Preview
+    def rollup
+      teacher = User.find_or_initialize_by(
+        name: 'Teacher MailerPreview',
+        email: 'teacher@rollup_mailer_preview.com',
+        role: 'teacher'
+      )
+
+      teacher_info = TeacherInfo.find_or_initialize_by(user: teacher)
```

Comment:
Note that we don't currently load FactoryBot into non `test` environments. Thus, I opted to initialize (not persist) records using normal AR syntax.

Reply:
Is `teacher_info` needed here?

Reply:
Yes - `see rollup_mailer.rb:32`